cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -226,11 +226,11 @@ func (sm *hardwareStateMachine) handleHardwareStartedEvent(event hardwareStarted
 func (sm *hardwareStateMachine) handleHardwareStartRequestEvent(event hardwareStartRequestEvent) {
 	sm.log("handling hardware start request event")
 	switch sm.currentState.(type) {
-	case *hardwareOff, *hardwareStopping:
+	case *hardwareOff, *hardwareStopping, *hardwareRestarting:
```

Comment:
I'm wondering if I had it the way it was because I thought a restart should complete (or fail) before we tried starting/stopping. Is the problem really that we're not coming out of the restarting state ?

Reply:
OK I've reminded myself of how this works; it does come out of the state using a hardwareStartRequestEvent.